docs/source/conf.py

```diff
@@ -53,7 +54,7 @@
 source_suffix = '.rst'
 
 # The encoding of source files.
-#source_encoding = 'utf-8-sig'
+# source_encoding = 'utf-8-sig'
```

Comment:
I'd recommend excluding `docs/source/conf.py`, the formatting changes don't really make it more readable.